.github/workflows/sdk-regression.yml

```diff
@@ -34,6 +34,15 @@ jobs:
       - uses: xt0rted/pull-request-comment-branch@v2
         if: ${{ github.event.issue.pull_request }}
         id: comment-branch
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: regex-match
+        with:
+          text: ${{ steps.comment-branch.outputs.head_ref }}
+          regex: '^[a-zA-Z0-9_/\-]+$'
```

Comment:
Please add same check in all child repos too
Reason: they run on workflow_dispatch, so anyone can trigger action in child repo manually

Reply:
This comment is not addressed

Reply:
Running workflow dispatch in the child repo needs our token

Reply:
check should be in both places, since shell injection is present in that repo, if run type is changed, the vulnerability can be exploited